Quantum-Control-Applications/Superconducting/Two-Fixed-Coupled-Transmons/18c_CR_calib_cr_drive_phase.py

```diff
@@ -1,12 +1,17 @@
 """
-                                 CR_calib_cr_drive_phase
+        CR_calib_cr_drive_phase
 
 The CR_calib scripts are designed for calibrating cross-resonance (CR) gates involving a system
 with a control qubit and a target qubit. These scripts help estimate the parameters of a Hamiltonian,
 which is represented as:
     H = I ⊗ (a_X X + a_Y Y + a_Z Z) + Z ⊗ (b_I I + b_X X + b_Y Y + b_Z Z)
 
-For the calibration sequences, we employ echoed CR drive.
+The sequence extracts the six coefficients as a function of CR drive phase,
+corresponding to Fig. 3(a) of the referenced paper.
+
+For the calibration sequences, we use one of the following CR drive configurations:
+"direct," "direct + echo," "direct + cancel," or "direct + cancel + echo."
+
```

Comment:
We can also say at the end that we need to update the phases no?

Reply:
Yes, says at `l.37`